presto-orc/src/test/java/com/facebook/presto/orc/FakeOrcDataSource.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import java.util.Map;
+
+class FakeOrcDataSource
```

Comment:
Given we will need a general test class, can we
- make `DummyOrcDataSource` to a top-level public class in test package `com.facebook.presto.orc`
- rename it to `NoopOrcDataSource`
- reuse `NoopOrcDataSource` in this commit as well as its original place for `TestListFilter` and `TestCachingOrcDataSource`?